.github/workflows/ci.yaml

```
@@ -0,0 +1,71 @@
+name: CI
+
+on: [workflow_dispatch, pull_request]
+
+env:
+  SECRET_KEY: thisisasecret
+  DATABASE_URL: postgresql://postgres:password@localhost:5432/postgres
+
+jobs:
+  ci:
+    strategy:
+      matrix:
+        python-version: ["3.6"]
+        platform: [ubuntu-20.04]
+
+    runs-on: ${{ matrix.platform }}
+
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+
+    - name: Set up Python dependencies
+      run: |
+        pip install -U pip
+        # pip install wheel
+        # pip wheel -q -r requirements/travis.txt
+        pip install -r requirements/travis.txt
```

nit: how about renaming the file?
```suggestion
        pip install -r requirements/ci.txt
```

---

resolved in 6086dc7e1b6a6776c961b563c5bb3d461e5cf540